src/lightning_app/cli/lightning_cli.py

```diff
@@ -29,10 +31,68 @@ def get_app_url(runtime_type: RuntimeType, *args) -> str:
 @click.group()
 @click.version_option(ver)
 def main():
-    register_all_external_components()
     pass
 
 
+@main.group()
+def show():
+    """Show info about resource."""
+    pass
+
+
+@show.command()
+@click.argument("app_name", required=False)
+@click.argument("components", nargs=-1, required=False)
+@click.option("-f", "--follow", required=False, is_flag=True)
+def logs(app_name: str, components: List[str], follow: bool) -> None:
```

Comment:
Docstrings and help strings will be nice. Specially for components to indicate the `flow` can be used